modules/ai_engine_embedding/src/Form/AiEngineEmbeddingSettings.php

```diff
@@ -67,6 +91,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('The chunk size to split each document into'),
       '#default_value' => $config->get('azure_chunk_size') ?? 3000,
     ];
+    $form['included_media_types'] = [
+      '#type' => 'checkboxes',
+      '#title' => $this->t('Included Media Types'),
+      '#options' => $this->getMediaTypes(),
+      '#default_value' => array_keys($config->get('included_media_types') ?? []),
+      '#default_value' => $config->get('included_media_types') ?? [],
```

Comment:
Highlander - there can only be one default value.

Reply:
HAHAHAHAHA OMG I'm so glad you saw that.  Too funny.  Fixing that one now.